test/test_network_constructors.jl

```diff
@@ -99,8 +103,6 @@ end
     ps_model = OperationsProblem(TestOpProblem, network, c_sys5; optimizer = GLPK_optimizer)
     construct_device!(ps_model, :Thermal, thermal_model)
     construct_device!(ps_model, :Load, load_model)
-    @test_throws ArgumentError construct_network!(ps_model, network)
```

Comment:
This negative test check is no longer relevant.  The code will fail to run unless you pass an instance of NetworkOperationsParameters.